packages/reserve-site/src/components/Footer.tsx

```diff
@@ -12,12 +12,25 @@ export default function Footer() {
           <strong>Disclaimer</strong> Nothing herein constitutes an offer to sell, or the
           solicitation of an offer to buy, any securities or tokens.
         </div>
+        <div css={copyRightStyle}>© {year} AP Reserve Foundation</div>
```

Comment:
Nit shouldn't the year just pull the current year from `Date`

Reply:
yeah that would probably work well aye